src/SingleBookingPage/__tests__/BookingDetail.test.js

```diff
@@ -58,6 +59,13 @@ describe('BookingDetail >', () => {
       expect(wrapper.find(Notification).props().isUrgent).toBeTruthy();
     });
 
+    it('is not displayed when status is anything but CONFIRMED', () => {
+      // $FlowExpectedError: We don't need all props for tests
+      const refundedBooking = { ...booking48HoursAgo, status: 'REFUNDED' };
```

Comment:
hmm maybe we could test for several different values of status here? I'm thinking of something like this:
```
['REFUNDED', 'CANCELLED', ...].forEach(status => it(`is not displayed when status is anything but ${status}`, () => {
  ... rest of the test ...
})
```

```

Reply:
🤔  we can do it, but does it worth? it's quite much for one condition

Reply:
Yeah, not sure :smiley:...